src/components/PostComment.jsx

```diff
@@ -0,0 +1,68 @@
+import { useContext, useState } from "react"
+import { postCommentByArticleId } from "../api"
+import { UserContext } from "../contexts/userContext"
+
+const PostComment = ({article_id, setComments}) => {
+    const [postComment, setPostComment] = useState("")
+    const [isError, setIsError] = useState(false)
+    const [isLoading, setIsLoading] = useState(false)
+    const {user} = useContext(UserContext)
+
+    const handleCommentInput = (event) => {
+        setPostComment(event.target.value)
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        
+        const newCommentInput = {
+            author: user.username,
+            body: postComment,
+            created_at: new Date().toISOString(),
+            votes: 0,
+        }
+
+        setIsLoading(true)
+        setTimeout(() => {
+            postCommentByArticleId(article_id, newCommentInput)
+            .then(() => {
```

Comment:
This anonymous CB will be invoked with the added comment, which will also have the unique id provided by SQL. We should use this response on line 29 instead of newCommentInput and then our comment would have an ID and deletion would be possible